src/Symfony/Component/Serializer/Context/Normalizer/DateTimeNormalizerContextBuilder.php

```diff
@@ -43,12 +43,15 @@ public function withFormat(?string $format): static
      *
      * @see https://secure.php.net/manual/en/class.datetimezone.php
      *
+     * @param ?bool $force If `true` will ignore any timezone defined
+     *                     in date strings and use the given timezone
+     *
      * @throws InvalidArgumentException
      */
-    public function withTimezone(\DateTimeZone|string|null $timezone): static
+    public function withTimezone(\DateTimeZone|string|null $timezone, ?bool $force = null): static
```

Comment:
```suggestion
    public function withTimezone(\DateTimeZone|string|null $timezone, bool $force = false): static
```

Reply:
I thought about setting it to `true|false` but in the implementation there's a "default context" and given context. Basicly you could set default context to `->withTimezone(null, true)`, and in your custom context `->withTimezone('UTC, null)`. In this way for _every_ context the timezone will be forced, but per serializer context you can enforce different timezones.